Box.V2/Models/BoxZipDownloadStatus.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Box.V2.Models.Request
+{
+    /// <summary>
+    /// Box representation of the status of a download for a zip file
+    /// </summary>
+    public class BoxZipDownloadStatus
+    {
+        public const string FieldTotalFileCount = "total_file_count";
+        public const string FieldDownloadedFileCount = "downloaded_file_count";
+        public const string FieldSkippedFileCount = "skipped_file_count";
+        public const string FieldSkippedFolderCount = "skipped_folder_count";
+        public const string FieldState = "state";
+
+        /// <summary>
+        /// The total number of files of the zip 
+        /// </summary>
+        [JsonProperty(PropertyName = FieldTotalFileCount)]
+        public int TotalFileCount { get; set; }
+
+        /// <summary>
+        /// The number of files in the zip that were downloaded
+        /// </summary>
+        [JsonProperty(PropertyName = FieldDownloadedFileCount)]
+        public int DownloadedlFileCount { get; set; }
+
+        /// <summary>
+        /// The number of files in the zip that were skipped when downloading
+        /// </summary>
+        [JsonProperty(PropertyName = FieldSkippedFileCount)]
+        public int SkippedFileCount { get; set; }
+
+        /// <summary>
+        /// The number of folders in the zip that were skipped when downloading
```

Comment:
```suggestion
        /// The number of folders in the zip that were skipped when creating the zip file
```

Reply:
According to Mohammed, "we might skip downloading a file/folder if you don't have no download access/permissions to it we prefer not to expose this information (i mean permission related information) as this might lead to information discloser
so we just say we skipped a file/folder and we don't mention no reason"